force_wfmanager/run.py

```diff
@@ -12,10 +13,19 @@
 from force_wfmanager.wfmanager_plugin import WfManagerPlugin
 
 from traits.api import push_exception_handler
+
+from .version import __version__
+
 push_exception_handler(lambda *args: None, reraise_exceptions=True)
 
 
+@click.command()
+@click.version_option(version=__version__)
 def main():
+    _main()
+
+
+def _main():
```

Comment:
Again, on the "naming things is hard" front, I think click expects the decorated function name to match the name of the command (unless explicitly overridden), so in this situation I think you might want something like:
```
@click_command(name='force-wfmanager')
@click.version_option(version=__version__)
def force_wfmanager():
    main()

def main():
    ...
```

Reply:
Also, another opportunity for a "drive-by" improvement: give the functions docstrings, even if just one-liners!

Reply:
Will do! For future stuff with more complex docstrings is there a particular style to follow?

Reply:
Yes - Numpy style: see https://numpydoc.readthedocs.io/en/latest/format.html and http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html